src/lib/sql/tables.sql

```diff
@@ -5,9 +5,9 @@ SELECT
   table_name AS name,
   is_insertable_into,
   is_typed,
-  pg_total_relation_size(table_schema || '.' || table_name) :: bigint AS bytes,
+  pg_total_relation_size(format('%I.%I', table_schema, table_name))::bigint AS bytes,
```

Comment:
can you help me to understand what this change does?

Reply:
If this is a breaking change, we shouldn't push it as the table sizes are used in Production already

Reply:
This is part of the fix in f1de03151879794ba9f73ddbe85ea5fb00ddc844 to quote identifiers with whitespace within the name. Creating a table with the name "foo bar", for example, will crash the server.

I figured I might as well quote everywhere I see identifiers are used since I find no reason not to.

Reply:
Nice one! very good catch